pkgs/applications/misc/ddcui/default.nix

```
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, wrapQtAppsHook
+, qtbase
+, qttools
+, ddcutil
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
mkDerivation rec {
```
Using `mkDerivation` directly as an input will pickup the version from Qt.

---

Thanks! Done